.env

```diff
@@ -1,10 +1,35 @@
+### Environment File ##################################################################
+# This is a simple configuration 
+# Online documentation for this file is available at 
+# https://docs.microsoft.com/en-us/dynamics365/commerce/e-commerce-extensibility/configure-env-file
+########################################################################################
+
+# MSDyn365_APP_TYPE variable is required. It can be set only to the value partner.
 MSDyn365_APP_TYPE=partner
+
+# PORT variable is required. It defines the port number that is used to preview your Node 
+#   application when the Node server is started by using the yarn start command. The default value is 4000
 PORT=4000
 
-### Partner Level Settings
-# MSDyn365_HOST=<Partner Host Here>
-# MSDyn365Commerce_BASEURL=<Retail Server Base URL Here>
-# MSDyn365Commerce_CHANNELID=<Retail Server Channel ID Here>
-# MSDyn365Commerce_CATALOGID=<Retail Server Catalog ID Here>
-# MSDyn365Commerce_OUN=<Retail Server OUN Here>
-# MSDyn365Commerce_BASEIMAGEURL=<Retail Server Base Image URL Here>
+# MSDyn365_HOST variable defines the domain name of your customer-facing e-Commerce site. 
+#   When this variable is set, if the URL https://localhost:4000/ is opened in a development environment, 
+#   your e-Commerce site will be rendered locally. If your site is protected through Azure Active Directory 
+#   (Azure AD) credentials, a prompt for a user name and password will appear.
+MSDyn365_HOST=
```

Comment:
Did you verify that an empty variable like this will be treated as if it's not set at all?